src/lamp_py/ingestion/convert_gtfs_rt.py

```diff
@@ -405,55 +402,90 @@ def write_local_pq(self, table: pyarrow.Table, local_path: str) -> None:
                 ]
```

Comment:
just seeing this now, but the line

`if self.sync_with_s3(local_path) is True:`

block can drop the `is True` portion.

Reply:
Incorporated